app/assets/javascripts/live_notify.js

```diff
@@ -1,37 +1,33 @@
 (function () {
-  var dispatcher = new WebSocketRails(host_name());
+  var dispatcher = new WebSocketRails(host_name())
```

Comment:
Is there a JS linter on this project? And why are we losing our line ending ';'.